docs/guides/postman-collection.md

```diff
@@ -0,0 +1,87 @@
+| digest             | http + digest scheme   |
+| oauth1             | http + bearer scheme   |
+| oauth2             | http + bearer scheme   |
+| awk                | Hardcoded header value |
+| aws                | Hardcoded header value |
+| akamai             | Hardcoded header value |
+| ntlm               | Hardcoded header value |
+
+## Events
+
+Events are linked to the Postman hosted platform, and so Prism is going to skip this section entirely.
+
+**Warning:** these scripts/event handlers could potentially modify variable values, with the consequence of changing the way the subsequent requests would work.
+
+## Request Body
+
+Postman Collections support saving GraphQL queries as well as regular HTTP. Prism has no GraphQL support at the moment, so that section is going to be skipped.
+
+In case the disabled flag is set to true for any of the parameter type, Prism will ignore such section, as there is no concept of disabled parameter.
+
+## JSON Schema Generation
+
+OpenAPI 2 and 3 are schema driven, while Postman Collections are examples driven.
+
+Essentially, while in OpenAPI 2/3 you can either define an example request/response body OR define a JSON Schema to describe how the payload will look like, in a Postman Collections you can only define examples.
+
+For such reason, Prism will _try_ to infer a JSON Schema from an example, and use such one to perform request/response validation as well as examples generation in case the it is running in `dynamic` mode.
+
+Infer a JSON Schema from a payload is an operation usually complicated — and there's no way a perfect result will be achieved. We can cover the base cases, but keep in mind you might get validation errors that are legit.
+
+To give an example, this JSON object:
+
+```json
+{
+  "name": "Bugatti Veyron",
+  "type": "Sport Car",
+  "speed": 3000
+}
+```
+
+will likely generate this schema:
+
+```json
+{
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "type": {
+      "type": "string"
+    },
+    "speed": {
+      "type": "number"
+    }
+  }
+}
+```
+
+Although the result seems good:
+
+- There is no way to understand whether all or some of the properties are required or not, so they'll all be marked as optionals.
+- There is no way to understand whether a type is indeed an `enum` instead of a regular `string`. In this specific example, maybe the author's intention is that type is either `Sport Car` or `SUV`
+- There is no way to understand whether a type can be something stricter; for example, maybe the author's intention was to mark the `speed` property as `integer` instead of `number`.
+
+## Operations Merging
+
+Postman Collections allows to define the same response multiple times, as long they differ for the returned response type, payload, examples. Prism will try to merge all these definitions in a single operation and then selecting the appropriate example based on its internal negotiator.
+
+## Reference documents
```

Comment:
This information was helpful in the design doc but I don't think it adds value to our users. We can delete this.

Reply:
Well, these docs may answer some questions like:
- what I did wrong in my PC so something is not picked up by Prism?
- what are the things I can keep in Postman Collection

I'd vote on keeping these references.

Reply:
Ok, if that's the use case then lets highlight that. Maybe we can have a "Debugging" header which says "Postman Collection support is still experimental, and you may notice some information is not being picked up, or things are not responding as expected. To help us figure out the problem, ..."

At this point im not sure how to glue the fact that this is end user documentation for Prism to links to explaining how Postman Collections are laid out. Why would they care? We can assume their Postman Collection is valid, so why are we telling them to look at documentation or a schema for postman collections? Do we want them to dig into the Postman schema, and try to debug what the issue is in http-spec?

Remember this documentation is addressing end users, who may not be developers, who may just have a Postman file and want to run a mock server. We want to write all documentation as though we are addressing these people, not addressing contributors, or Stoplight employees. 

Also, if we want to help end users debug stuff, there might be better resources? Maybe tell them to look at Priskm logs if they're helpful? If they're not helpful, should we make them helpful?

Reply:
I'll remove it — too much discussion for two lines aren't worth :)